Assets/Scripts/Utility/ImageReader.cs

```diff
@@ -371,6 +371,17 @@ public static ImageData GetImageData(string filename, int record = 0, int frame
 
                     break;
 
+                case ImageTypes.JSON:
+                    string text;
+                    AssetInjection.TextAssetReader.TryRead(filename, out text);
```

Comment:
Just a general question about the general need for the new archive format.
Do I understand that this format allows you to take the image data of another archive while using new textures? Seems kind of orthogonal to the feature, but I suppose it's useful

Reply:
That's exactly the case. I was trying to find a way to make new image files with all the data required (offsets, sizes etc). After a few experiments, i found this solution to be the easiest for modding